command/sub8_missions/nodes/move_command

```diff
@@ -37,14 +47,53 @@ def main(args):
         fprint("Waiting for odom...")
         yield sub.tx_pose()
         fprint("Odom found!", msg_color='green')
-
+        rospack = rospkg.RosPack()
         action_kwargs = {'speed': float(args.speed), 'blind': bool(args.blind)}
 
         if command == 'custom':
             # Let the user input custom commands, the eval may be dangerous so do away with that at some point.
             fprint("Moving with the command: {}".format(argument))
             res = yield eval("sub.move.{}.go()".format(argument, **action_kwargs))
 
+        elif command in ['tp', 'teleport']:
+            try:
+                state_set_pub = yield nh.advertise('gazebo/set_model_state', ModelState)
+                config_file = os.path.join(rospack.get_path('sub8_gazebo'), 'config', 'teleport_locs.yaml')
+                f = yaml.load(open(config_file, 'r'))
+                try:
+                    x = float(argument.split(' ')[0])
+                    y = float(argument.split(' ')[1])
+                    z = float(argument.split(' ')[2])
+                except Exception:
```

Comment:
So in my current iteration on my comp, I have added checks for the types of exceptions that will directly error out the commands being run, namely: IndexError, ValueError, and LookupError. At the end of this entire block of code though is a final catch, in the event we run into an unexpected error. Is that safe to have or should I just make that final catch a KeyboardInterrupt Exception and terminate there. 
Also as far as termination goes, I am using the "break" keyword. Is there something better I should use?